### PR TITLE
DOC-9902 Add backup/restore enhancements to what's new

### PR DESCRIPTION
**Pull request by mattyhall**

### CODE REVIEW

**Comment by mattyhall** [comment]
I jumped the gun on this a little, sorry - can we hold off merging for a few days please

**Comment by mattyhall** [comment]
I think this change is good to be reviewed now

File: modules/introduction/pages/whats-new.adoc [hunk 1]
```diff
@@ -117,3 +117,20 @@ For information, see xref:learn:clusters-and-availability/rebalance.adoc#smart-b
 
 * _Array Flattening_, whereby the elements of arrays can be more efficiently indexed.
 See xref:n1ql:n1ql-language-reference/metafun.adoc#flatten_keys[FLATTEN_KEYS].
+
+=== Backup/Recovery Enhancements
+
+Directory Backup to Google Cloud Platform (GCP) Cloud Storage::
```
Comment (jamesl33) [inline modules/introduction/pages/whats-new.adoc]: > Directory Backup

Should this be "Direct Backup"?

File: modules/introduction/pages/whats-new.adoc [hunk 2]
```diff
@@ -117,3 +117,20 @@ For information, see xref:learn:clusters-and-availability/rebalance.adoc#smart-b
...
+The cbbackupmr CLI and the Backup Service now support direct backup to GCP cloud storage.  See xref:backup-restore:cbbackupmgr-cloud.adoc[]
```
Comment (jamesl33) [inline modules/introduction/pages/whats-new.adoc]: > now support

Should this be "now supports"

**Review by tonyjhillman — APPROVED** [review]

**Pull request merged**